src/styles/var.scss

```
@@ -0,0 +1,12 @@
+$cell-size: 100px;
+$gap-size: 1px;
+$border-size: 1px;
+$week-days-position: (
+  "mon": 0,
+  "tue": ($cell-size + $gap-size),
+  "wed": 2 * ($cell-size + $gap-size),
+  "thu": 3 * ($cell-size + $gap-size),
+  "fri": 4 * ($cell-size + $gap-size),
+  "sat": 5 * ($cell-size + $gap-size),
+  "sun": 6 * ($cell-size + $gap-size)
```

would be better to move  ($cell-size + $gap-size) this part to loop directly for avoid repeating 
```suggestion
  "mon": 0,
  "tue": 1,
  "wed": 2 ,
  "thu": 3 ,
  "fri": 4 ,
  "sat": 5 ,
  "sun": 6 ,
```